app/poros/i_scream_results.rb

```diff
@@ -112,4 +112,22 @@ def new_game_params(data)
       video:              data[:video]
     }
   end
+
+  def games_by_keywords(keywords)
+    results = service.get_games_by_keywords(keywords)
+    return results if results.is_a?(String)
+    results[:data].map do |api_return|
+      data = api_return[:data]
+      game = game_object(data)
+
+      category_objects(data, game)
```

Comment:
Each one of these methods generates the model in the title, see below for example

Reply:
This looks really nice